ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/services/DefaultKsqlClient.java

```diff
@@ -154,4 +161,15 @@ private static HttpClientOptions createClientOptions() {
     return new HttpClientOptions().setMaxPoolSize(100);
   }
 
+  private static Map<String, String> toClientProps(final KsqlConfig ksqlConfig) {
+    final Map<String, String> clientProps = new HashMap<>();
+    for (Map.Entry<String, Object> entry : ksqlConfig.originals().entrySet()) {
+      if (entry.getValue() instanceof String) {
```

Comment:
is there any reason we ignore the non-string properties? (should we be calling toString on them?)

Reply:
we're really only interested in the ssl config values which are all strings I think.

Reply:
I'm worried that assumption is a little brittle, at a minimum can we add a comment explaining that?

Reply:
Ok, I'll toString the value :)